com.unity.multiplayer.mlapi/Editor/NetworkManagerEditor.cs

```diff
@@ -167,6 +167,24 @@ private void OnEnable()
         m_NetworkPrefabsList = new ReorderableList(serializedObject, serializedObject.FindProperty(nameof(NetworkManager.NetworkConfig)).FindPropertyRelative(nameof(NetworkConfig.NetworkPrefabs)), true, true, true, true);
         m_NetworkPrefabsList.drawElementCallback = (Rect rect, int index, bool isActive, bool isFocused) =>
         {
+            for (int i = 0; i < m_NetworkManager.NetworkConfig.NetworkPrefabs.Count; i++)
+            {
+                // Find the first playerPrefab
+                if (m_NetworkManager.NetworkConfig.NetworkPrefabs[i].PlayerPrefab)
+                {
+                    // Iterate over all other and set player prefab to false
+                    for (int j = 0; j < m_NetworkManager.NetworkConfig.NetworkPrefabs.Count; j++)
+                    {
+                        if (j != i && m_NetworkManager.NetworkConfig.NetworkPrefabs[j].PlayerPrefab)
+                        {
+                            m_NetworkManager.NetworkConfig.NetworkPrefabs[j].PlayerPrefab = false;
+                        }
+                    }
+
+                    break;
+                }
+            }
+
```

Comment:
so, if we're registering for `drawElementCallback` event, how frequently this method body will be called? every frame/draw or on change/toggle?

Reply:
It's kindof undocumented. There is no official documentation for ReorderableList. But I think it's when something changes.